### DISCUSSION TITLE
output jupyter notebook is empty

### DISCUSSION THREAD

**Discussion by sheng19331** [KosinskiLab/AlphaPulldown#383] [Ideas]
Hi 

Thanks for the nice tool. 
I tried to run the create_notebook.py in the output folder with an indication of 'notebook has been successfully created'. But the notebook has no results displayed. The differen thing is the output i got is by running the colabfold-local package instead of  ALphaPulldown. Would this affect the output of creating notebook?  But i do see it is processing or scanning the results from the terminal after running the create_notebook.py. The cutoff value has also been adjusted in case it is too strict, but still the output is empy. 

Best
Sheng

**Comment by dingquanyu** [comment]
Hi,

Thanks for your interest in our package. Unfortunately, this script is meant for the result folders generated by AlphaPulldown. I haven't use colabfold-local myself so I don't really know what the results look like but I wouldn't expect the script to be compatible.

Yours
Dingquan

**Comment by dingquanyu** [comment]
I will convert this to discussion as it's not directly related to the codes.
